platforms/core-runtime/build-configuration/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencies {
 
     implementation(project(":base-services"))
     implementation(project(":logging"))
+    implementation(project(":daemon-protocol"))
```

Comment:
Inverted the relationship between projects. This project contains tasks for upgrading the build JVM, so it should depend on lower level projects, rather than the other way around.